src/renderer/services/utils/account.ts

```diff
@@ -26,6 +31,22 @@ export const deleteKeystore = (directory: string, publicKey: string): void => {
     selectedV4Keystore.destroy();
 };
 
+export const deleteBeaconNodeContainers = async(): Promise<void> => {
+    // Remove containers
+    await Promise.all(networks.map(async(network) => {
+        if (network.networkName !== "localhost") {
+            await DockerRegistry.removeContainerPermanently(network.dockerConfig.name);
```

Comment:
You should probably catch errors so we don't leave hanging containers in case of error